package.json

```diff
@@ -123,6 +123,8 @@
     "typescript": "^4.5.5"
   },
   "resolutions": {
+    "**/istanbul/async": "^2.6.4",
```

Comment:
portfinder/async?

Reply:
not necessary, webpack-dev-server gets it from portfinder which gets it from something else down the chain until eventually it comes from `jake`

Reply:
After this PR:

```
dev-dsk-xss-1d-cefd8e9f % yarn why async
yarn why v1.18.0
[1/4] Why do we have the module "async"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "async@2.6.4"
info Has been hoisted to "async"
info Reasons this module exists
   - "workspace-aggregator-24761829-0968-405a-87cb-9a796c11e360" depends on it
   - Hoisted from "_project_#amplify-velocity-template#istanbul#async"
   - Hoisted from "_project_#amplify-graphiql-explorer#webpack-dev-server#portfinder#async"
   - Hoisted from "_project_#@aws-amplify#cli-internal#ejs#jake#async"
   - Hoisted from "_project_#amplify-graphiql-explorer#workbox-webpack-plugin#workbox-build#@surma#rollup-plugin-off-main-thread#ejs#jake#async"
info Disk size without dependencies: "852KB"
info Disk size with unique dependencies: "5.71MB"
info Disk size with transitive dependencies: "5.71MB"
info Number of shared dependencies: 1
=> Found "winston#async@3.2.3"
info This module exists because "_project_#amplify-cli-logger#winston" depends on it.
info Disk size without dependencies: "1.08MB"
info Disk size with unique dependencies: "1.08MB"
info Disk size with transitive dependencies: "1.08MB"
info Number of shared dependencies: 0
=> Found "archiver#async@3.2.3"
info This module exists because "_project_#amplify-category-function#archiver" depends on it.
info Disk size without dependencies: "1.08MB"
info Disk size with unique dependencies: "1.08MB"
info Disk size with transitive dependencies: "1.08MB"
info Number of shared dependencies: 0
=> Found "bestzip#async@3.2.3"
info This module exists because "_project_#@aws-amplify#graphql-predictions-transformer#bestzip" depends on it.
info Disk size without dependencies: "1.08MB"
info Disk size with unique dependencies: "1.08MB"
info Disk size with transitive dependencies: "1.08MB"
info Number of shared dependencies: 0
Done in 1.71s.
```